src/components/Table/Table.spec.js

```diff
@@ -54,6 +54,18 @@ describe('Table', () => {
       const actual = create(<Table headers={headers} rows={items} condensed />);
       expect(actual).toMatchSnapshot();
     });
+
+    it('should render a scrollable table', () => {
+      const actual = create(
+        <Table headers={headers} rows={items} scrollable rowHeaders={false} />
+      );
+      expect(actual).toMatchSnapshot();
+    });
+
+    it('should not render a scrollable table if the rowHeaders prop is true', () => {
+      const actual = create(<Table headers={headers} rows={items} />);
+      expect(actual).toMatchSnapshot();
+    });
```

Comment:
I'm missing some functional tests, but I'm also not sure how you could test the scrolling behavior. Seems to be more suited for a browser test 🤔